### PR TITLE
revisions to style and content of claim_revision_member template

### PR DESCRIPTION
**Pull request by Schparky**
image
image

### CODE REVIEW

File: application/templates/mail/claim_revision_member.plush.html [hunk 1]
```diff
@@ -1,41 +1,38 @@
...
+		<%= claim.StatusReason %>
 	</p>
 
 	<p>
-		<%= supportFirstName %> says:
-	</p>
-
-	<p style="text-indent:2em;">
-		<%= claim.StatusReason %>
+		&mdash; <%= supportFirstName %>
```
Comment (Baggerone) [inline application/templates/mail/claim_revision_member.plush.html:22]: The Figma doesn't include a space after the dash. (I had to double check.)

File: application/templates/mail/_claim_card.plush.html [hunk 2]
```diff
@@ -1,30 +1,46 @@
-
-<div>
-	<h4 style="margin:0; padding:0">Claim ID</h4>
-	<%= claim.ReferenceNumber %>
+<div style="
+			padding: 16px;
```
Comment (Baggerone) [inline application/templates/mail/_claim_card.plush.html:2]: If you're making other changes anyway, I guess you could reduce the indentation on these.

File: application/templates/mail/_claim_card.plush.html [hunk 3]
```diff
@@ -1,30 +1,46 @@
-
-<div>
-	<h4 style="margin:0; padding:0">Claim ID</h4>
-	<%= claim.ReferenceNumber %>
+<div style="
```
Comment (Baggerone) [inline application/templates/mail/_claim_card.plush.html:1]: The spacing in the email above this section seems a bit too much.

**Review by Baggerone — APPROVED** [review]
Just cosmetic things. I don't know if you want to ignore those for now.

**Comment by Schparky** [comment]
fixing the style things on the next PR

**Pull request merged**